avocado/core/parameters.py

```diff
@@ -0,0 +1,306 @@
+# This program is free software; you can redistribute it and/or modify
```

Comment:
Wouldn't `test_params.py` be more suitable? I mean in `avocado.core` the `parameters` could mean anything and my first guess would be either `runner args` or some `runner parameters`.

Reply:
Also if we called it `test_params` the actual class could be `test_params.Params` instead of `AvocadoParams` as the `Avocado` part is kind-of duplicitous.